src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -119,13 +119,15 @@ export class FileExplorerComponent implements OnInit {
   }
 
   public goRoot() {
+    BucketService.isResponseComplete = false;
```

Comment:
This is a bit ugly, as the code is reusing the bucket service property to track a different state, the state of the `getUserFirecloudWorkspaces()` call. It would be better for this class to have its own field and then define `isResponseComplete()` below as the combination of both states.